faiss/IndexRefine.cpp

```diff
@@ -54,6 +54,14 @@ void IndexRefine::add(idx_t n, const float* x) {
     ntotal = refine_index->ntotal;
 }
 
+size_t IndexRefine::remove_ids(const IDSelector& sel) {
+    FAISS_THROW_IF_NOT(is_trained);
+    base_index->remove_ids(sel);
+    refine_index->remove_ids(sel);
+    ntotal = refine_index->ntotal;
+    return ntotal;
```

Comment:
Sorry, let me clarify. The return value should be the number of vectors removed, not the number of vectors remaining.
```suggestion
    ntotal_before = refine_index->ntotal
    base_index->remove_ids(sel);
    refine_index->remove_ids(sel);
    ntotal_after = refine_index->ntotal;
    return ntotal_after - ntotal_before;
```